tests/robot-tests/tests/libs/admin-common.robot

```diff
@@ -569,7 +573,7 @@ user navigates to admin dashboard
     user navigates to admin dashboard if needed    %{ADMIN_URL}
     user waits until h1 is visible    Dashboard
     IF    "${USER}" != ""
-        user waits until page contains title caption    Welcome ${USER}
+        user waits until page contains title caption    Welcome ${USER}    10
```

Comment:
Same with the other comment - I think the 10 here actually reduces the wait time from 30.